BiasAndFairnessServers/src/database/db.py

```
@@ -0,0 +1,22 @@
+import dotenv
+dotenv.load_dotenv()
+
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+from .config import settings
```

Pylint (3.3.7)

[error] Attempted relative import beyond top-level package

(E0402)